### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by flaviooliveira-code**
Correção da escrita do comando "origin"

### CODE REVIEW

**Review by badyl562 — APPROVED** [review]

**Comment by Desynkova** [comment]
Good

**Comment by Desynkova** [comment]
bc1qkz2e0vn23a9c2lrtf0klwdjctjerzkesjtleer

**Review by thepowerzero — APPROVED** [review]

**Comment by MIKHAIL3942789M** [comment]
> f are 

Mikhail

**Comment by MIKHAIL3942789M** [comment]
Merci

**Comment by MIKHAIL3942789M** [comment]
VideoCapture_20210614-153803
Mvdi(xnt}235}><>$854<>☆}{]87}

**Review by Russell4115 — APPROVED** [review]

**Review by Russell4115 — APPROVED** [review]

**Review by Russell4115 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Esse comando além de criar a branch já entra nela com o checkout, inclusive se
 
 * E agora fazemos o passo a passo que já sabemos, colocamos a nossa alteração em stagging com o `git add .` e commitamos com o `git commit -m "novo botão"`
 
-* Para enviarmos agora que vai ser diferente. Vocês lembram que utilizávamos o `git push orgin main` né? Porém main era aquela branch principal. Agora então usaremos `git push origin botao`
```
Comment (Phyo4) [inline README.md:109]: Game player

**Comment by Phyo4** [comment]
ML Player

**Review by GabrielVitorGL — APPROVED** [review]

**Pull request closed without merging**